frontend/src/common/MultiFileInput.tsx

```diff
@@ -80,7 +80,7 @@ export default function MultiFileInput({
   )
 
   return (
-    <Box sx={{ ...(fullWidth && { width: '100%' }) }}>
+    <Box sx={[fullWidth && { width: '100%' }]}>
```

Comment:
This doesn't look right to me. I have a feeling a codemod didn't like our syntax here. Perhaps try:
```
sx={[ fullWidth ? { width: '100%' } : {} ]}
```